src/app/core/components/breadcrumbs/breadcrumbs.component.scss

```diff
@@ -1,16 +1,15 @@
 .breadcrumb-container {
```

Comment:
You should style the breadcrumbs according to the mockup.
The separator has to be blue. The active route has to be **bold**. Add margins to the items.
You can use BEM here. It will be more convinient.